crowd_anki/representation/json_serializable.py

```diff
@@ -6,8 +6,8 @@
 
 class JsonSerializable:
     readable_names = {}
-    export_filter_set = {"mod",  # Modification time
-                         "usn",  # Todo clarify
+    export_filter_set = {  # "mod",  # Modification time
```

Comment:
don't forget to revert hotfix changes :)

Reply:
Of course :wink:

Reply:
Done.